src/test/java/core/basesyntax/service/impl/QuantityServiceImplTest.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.QuantityService;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class QuantityServiceImplTest {
+    private static final Map<Fruit, Integer> ACTUAL = Storage.fruits;
```

Comment:
You don’t need to reassign Storage to a new variable. It’s  static so you can use it directly  but don't forget  clear storage after testing